modules/applications/pages/ui-builder/how-to-configure-https.adoc

```diff
@@ -0,0 +1,141 @@
+= HTTPS Configuration
+:page-aliases: applications:how-to-configure-https.adoc
+:description: In production, it is necessary to set up both Bonita Runtime and UI Builder for HTTPS instead of HTTP. The easiest way it to configure Bonita UI Proxy for SSL offloading.
+
+{description}
+
+Configuring HTTPS for Bonita Runtime and UI Builder can be achieved just by updating the nginx configuration of Bonita UI proxy and setting some environment variables. This page contains some configuration  examples.
+
+== Prerequisites
+
+To configure HTTPS, you need to obtain a certificate issued by a certificate authority for our domain. It is also advised to create a Diffie-Hellman parameters file (dhparam.pem) for increased security.
```

Comment:
How to?

Reply:
It depends on your operating system. This is external to Bonita and something that a devops should know, but I will add an example